### PR TITLE
Update what constitutes an active user.

### PR DESCRIPTION
**Pull request by teesloane**
Request to update what constitutes an active user; I wasn't sure if this should be added to the list or the sentence introducing the list — happy to change or leave it either way!

### CODE REVIEW

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -299,8 +299,9 @@ non-OSS projects. This includes activities such as:
 
 - Commits from users that trigger builds, including PR Merge commits.
 - Re-running jobs in the CircleCI web application, including [SSH debug]({{ site.baseurl }}/2.0/ssh-access-jobs).
-- Approving [manual jobs]({{ site.baseurl }}/2.0/workflows/#holding-a-workflow-for-a-manual-approval) (approver will be considered the actor of all downstream
-  jobs).
+  - Approving [manual jobs]({{ site.baseurl }}/2.0/workflows/#holding-a-workflow-for-a-manual-approval) (approver will be considered the actor of all downstream jobs).
+  - Using scheduled workflows
+  - Using the machine executor
```
Comment (emmabukacek) [inline jekyll/_cci2/faq.md]: Why do we call out `machine executor` here? Also, we'll probably want to include `Guest user`s, i.e. a user who reruns a build but doesn't have an equivalent Github user.

**Comment by emmabukacek** [comment]
Oh, and FYI, we maintain all Plans domain documentation here: https://docs.google.com/document/d/1Dt1Jzxz4X5qoIcDL6JjH-zriYxu91pMu5DgNtr59MSk/edit#heading=h.10l4d2xqm7jk

If there's something that isn't fleshed out, please bother us and we'll be happy to update it. :smile: Unfortunately it doesn't get the care it deserves, but we try and update it as often as possible.

File: jekyll/_cci2/faq.md (hunk 1 above)
Comment (teesloane) [reply jekyll/_cci2/faq.md]: I think I misunderstood the request, going to DM and clear up.

Comment (sararead) [reply jekyll/_cci2/faq.md]: @emmabukacek -- For `guest user`, could we be clearer with this language? Customers see multiple "guest users" on the account but do not understand what this is connected to. Is it API calls, scheduled workflows, users from open source projects?

**Comment by michelle-luna** [comment]
thank you!

**Pull request merged**